README.md

```diff
@@ -1,6 +1,6 @@
 # Code Climate ESLint Engine
 
-[![Code Climate](https://codeclimate.com/repos/55841b7a6956801212006c92/badges/92d8261f1b6200f19af5/gpa.svg)](https://codeclimate.com/repos/55841b7a6956801212006c92/feed)
+[![Code Climate](https://codeclimate.com/repos/57f285a969744e0d830009a7/badges/5ac270c5078729d2b4e1/gpa.svg)](https://codeclimate.com/repos/57f285a969744e0d830009a7/feed)
```

Comment:
can we use the OSS format of this link? E.g.: https://github.com/codeclimate/codeclimate-eslint/blob/master/README.md